### PR TITLE
Show courses of user

### PR DESCRIPTION
**Pull request by hoangmirs**

### CODE REVIEW

**Comment by ducfilan** [comment]
ok

File: app/views/admin/users/_show.html.erb [hunk 1]
```diff
@@ -1,64 +1,61 @@
...
-                <i class="glyphicon glyphicon-remove"></i>
-                <%= t "admin.users.index.btn_Delete" %>
-              <% end %>
-            <% end %>
-          </span>
-        </div>
+      </div>
+      <div class="panel-footer">
+        <%= mail_to @user.email, class: "btn btn-sm btn-primary" do %>
+        <i class="glyphicon glyphicon-envelope"></i>
```
Comment (thangtx88) [inline app/views/admin/users/_show.html.erb]: ident

File: app/views/admin/users/_show.html.erb [hunk 2]
```diff
@@ -1,64 +1,61 @@
...
+        <%= t ".send_mail" %>
```
Comment (thangtx88) [inline app/views/admin/users/_show.html.erb]: ident

File: app/views/admin/users/_show.html.erb [hunk 3]
```diff
@@ -1,64 +1,61 @@
...
+        <% end %>
+        <span class="pull-right">
+        <% if current_user?(@user) || @user.trainee? %>
+        <%= link_to edit_admin_user_path(@user),
```
Comment (thangtx88) [inline app/views/admin/users/_show.html.erb]: ident

File: app/views/admin/users/_show.html.erb [hunk 4]
```diff
@@ -1,64 +1,61 @@
...
+          class: "btn btn-sm btn-warning" do %>
+        <i class="glyphicon glyphicon-edit"></i>
+        <%= t "admin.users.index.btn_Edit" %>
```
Comment (thangtx88) [inline app/views/admin/users/_show.html.erb]: ident

File: app/views/admin/users/_show.html.erb [hunk 5]
```diff
@@ -1,64 +1,61 @@
...
+      <div class="panel-footer">
+        <%= mail_to @user.email, class: "btn btn-sm btn-primary" do %>
+        <i class="glyphicon glyphicon-envelope"></i>
+        <%= t ".send_mail" %>
+        <% end %>
+        <span class="pull-right">
+        <% if current_user?(@user) || @user.trainee? %>
+        <%= link_to edit_admin_user_path(@user),
+          class: "btn btn-sm btn-warning" do %>
+        <i class="glyphicon glyphicon-edit"></i>
```
Comment (thangtx88) [inline app/views/admin/users/_show.html.erb]: ident

File: app/views/admin/users/_show.html.erb [hunk 6]
```diff
@@ -1,64 +1,61 @@
...
+        <%= t "admin.users.index.btn_Edit" %>
+        <% end %>
+        <% end %>
+        <% if @user.trainee? %>
+        <%= link_to admin_user_path(@user), method: :delete, data:
```
Comment (thangtx88) [inline app/views/admin/users/_show.html.erb]: ident

**Pull request merged**